core/src/com/unciv/logic/automation/unit/CityLocationTileRanker.kt

```diff
@@ -124,7 +123,7 @@ object CityLocationTileRanker {
         }
 
         // Placing cities on the edge of the map is bad, we can't even build improvements on them!
-        tileValue -= (HexMath.getNumberOfTilesInHexagon(3) - tiles) * 2.4f
+        tileValue -= (19 - tiles) * 3f
```

Comment:
I don't like this, we're missing context on what 19 means 
Also, why 3?

Reply:
Ah, a case for C++'s [`constexpr`](https://github.com/Redrield/constexpr-kt) - and yeah changing the arbitrary-looking 2.4f for a different arbitrary-looking constant in a PR aiming to make another arbitrary-looking constant moddable... I bet you thought about moving that to ModConstants too, but couldn't think of a name...

Reply:
> I don't like this, we're missing context on what 19 means

It's described in the comment above, 19 being the third centered hexagonal number (1-7-19-37).

> Also, why 3?

To increase AI border avoidance: https://discord.com/channels/586194543280390151/618491418859798539/1431288119092707420. Maybe should go into a different PR though, I'll remove it from this one.